network-api/networkapi/wagtailpages/templates/wagtailpages/research_detail_page.html

```diff
@@ -2,67 +2,122 @@
 {% load wagtailcore_tags wagtailimages_tags %}
 
 {% block subcontent %}
-  <div>
-    {% image page.cover_image fill-256x348 as cover_image %}
-    <img src="{{ cover_image.url }}" alt="" class="tw-w-full tw-object-cover tw-object-center tw-max-w-[256px]"/>
-  </div>
-  <h1>
-    {{ page.title }}
-  </h1>
-  <div>
-    {% spaceless %}
-      {% if page.original_publication_date %}
-        <span>{{ page.original_publication_date }}</span>
-        {% if page.research_authors or page.related_topics %}
-          <span>|</span>
-        {% endif %}
-      {% endif %}
-      {% if page.research_authors %}
-        {% for research_author in page.research_authors.all  %}
-          <span>{{ research_author.author_profile.name }}{% if not forloop.last %}, {% endif %}</span>
-        {% endfor %}
-        {% if page.related_topics %}
-          <span>|</span>
-        {% endif %}
-      {% endif %}
-      {% if page.related_topics %}
-        {% for related_topic in page.related_topics.all  %}
-          <span>{{ related_topic.research_topic.name }}{% if not forloop.last %}, {% endif %}</span>
-        {% endfor %}
-      {% endif %}
-    {% endspaceless %}
-  </div>
-  {% if page.introduction %}
-    <p>{{ page.introduction }}</p>
-  {% endif %}
-  {% if page.research_links %}
-    <div>
-      {% for research_link in page.research_links.all  %}
-        <a href="{{ research_link.get_url }}">{{ research_link.label }}</a>
-      {% endfor %}
-    </div>
-  {% endif %}
-  {% if page.overview %}
-    <div>
-      <h2>Overview</h2>
-      {{ page.overview|richtext }}
-    </div>
-  {% endif %}
-  {% if page.research_authors %}
-    <div>
-      <h2>Project leads</h2>
-      {% for research_author in page.research_authors.all  %}
-        <div>
-          <h3>{{ research_author.author_profile.name }}</h3>
-          <p>{{ research_author.author_profile.tagline }}</p>
+
+  {% image page.cover_image fill-1240x144 as hero_img %}
+
+  <div class="tw-bg-no-repeat tw-bg-center tw-bg-cover tw-min-h-[144px]" style='background-image: url("{{ hero_img.url }}")'></div>
+
+
+  <div class="tw-max-w-6xl tw-mx-auto tw-pb-[100px] tw--mt-7 tw-bg-white tw-px-5 tw-pt-5">
+    <div class="medium:tw-grid tw-grid-cols-5 tw-grid-rows-1 tw-grid-flow-col tw-gap-7">
+      <div class="tw-col-span-1 tw-pt-7">
+        {# TODO Add image #}
+        TODO: Add image
+        {% comment %}
+        {% image page.image fill-259x344 as img %}
```

Comment:
Probably best done with a method on the page to retrieve the parent (library) banner image.